api/src/services/payload.ts

```diff
@@ -115,6 +116,19 @@ export class PayloadService {
 			if (Array.isArray(value)) return value.join(',');
 			return value;
 		},
+		async 'storage-public-url'({ action, payload }) {
+			if (action !== 'read') return;
+			const storage_config = getConfigFromEnv(`STORAGE_${payload.storage.toUpperCase()}_`);
+			if (!storage_config.publicUrl) return `${process.env.PUBLIC_URL}/assets/${payload.id}`;
```

Comment:
Wondering if we should just return "null" if it isn't configured 🤔 The public url should remain the external one imo. Maybe we should even call it "externalURL" instead of "publicURL"

Reply:
updated to return null if not configured. 

I can refactor the name as well if desired.